rfcs/accessibility/accessibility.md

```diff
@@ -0,0 +1,74 @@
+# RFC: Accessibility
+
+---
+
+Contributors:
+@jurokapsiar
+
+## Summary
+
+Fluent UI components are designed to support various aspects of accessibility, so that they can be used with different input methods (mouse, touch, keyboard, screen readers) as well as fulfill different rendering and layout requirements (theming, zoom, contrast).
+
+## Background
+
+[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/) and [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.2/) are community driven standard for accessible web pages and applications. They provide basic framework allowing consistent usage of web pages and applications with or without assistive technologies.
+
+## Problem statement
+
+Fluent UI components aim to fully respect both WCAG and ARIA and the Fluent UI team is working with the community to improve them. From the technical point of view, thee main challenges were identified:
+
+1. Accessibility is usually not intuitive as it is not immediately obvious. It is hard to achieve high quality user experience and easy to regress it.
+2. Each screen reader and browser works differently, often making it hard or even impossible to achieve high quality user experience for all combinations. Browsers and screen readers release new versions that change the experience
```

Comment:
`that change` not sure what would be the best but I would rather express this "can" happen, or it is high risk that new releases can bring regression....

Reply:
I agree, maybe something in the lines of

`One of the biggest reasons behind this is that browsers and screen readers are often updated, with each update having the potential to change the user experience from one version to the next.`